packages/html-reporter/src/filter.ts

```diff
@@ -144,7 +151,14 @@ export class Filter {
       if (!matches)
         return false;
     }
-
+    if (this.annotations.length) {
```

Comment:
This will return all the tests w/o annotations as matching the search criteria.

Reply:
I tried and it did not . Only test case with annotations will be returned . In below example ,only two test cases which had some annotation is displayed . Am I missing something here?
<img width="1078" alt="Screenshot 2024-03-28 at 7 28 31 PM" src="https://github.com/microsoft/playwright/assets/150931175/860d7db6-4c20-4248-bdb5-a6531e7bc679">

Reply:
Ah, you are right, I missed that `this` is a filter here. Still need a test for every new feature though.

Reply:
I have added the test cases for copy feature . Please review .